Source/WebCore/page/EditorClient.h

```diff
@@ -88,7 +92,7 @@ class EditorClient : public CanMakeWeakPtr<EditorClient>, public CanMakeCheckedP
     virtual void registerAttachments(Vector<SerializedAttachmentData>&&) { }
     virtual void registerAttachmentIdentifier(const String& /* identifier */) { }
     virtual void cloneAttachmentData(const String& /* fromIdentifier */, const String& /* toIdentifier */) { }
-    virtual void didInsertAttachmentWithIdentifier(const String& /* identifier */, const String& /* source */, bool /* hasEnclosingImage */) { }
+    virtual void didInsertAttachmentWithIdentifier(const String& /* identifier */, const String& /* source */, AttachmentAssociatedElementType /* associatedElementType */) { }
```

Comment:
Nit; the comment doesn't add much info, but I see that's the precedence in this file.